cmd/acmesolver/app/app.go

```diff
@@ -24,32 +24,30 @@ import (
 	"github.com/spf13/cobra"
 	"k8s.io/component-base/logs"
 
-	"github.com/cert-manager/cert-manager/internal/cmd/util"
 	"github.com/cert-manager/cert-manager/pkg/issuer/acme/http/solver"
 	logf "github.com/cert-manager/cert-manager/pkg/logs"
 )
 
-func NewACMESolverCommand(stopCh <-chan struct{}) *cobra.Command {
+func NewACMESolverCommand(_ context.Context) *cobra.Command {
 	s := new(solver.HTTP01Solver)
 	logOptions := logs.NewOptions()
 
 	cmd := &cobra.Command{
 		Use:   "acmesolver",
 		Short: "HTTP server used to solve ACME challenges.",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			rootCtx := util.ContextWithStopCh(context.Background(), stopCh)
+			runCtx := cmd.Context()
 
 			if err := logf.ValidateAndApply(logOptions); err != nil {
 				return fmt.Errorf("error validating options: %s", err)
 			}
 
-			rootCtx = logf.NewContext(rootCtx, logf.Log, "acmesolver")
-			log := logf.FromContext(rootCtx)
+			log := logf.FromContext(runCtx)
```

Comment:
Originally the logger was using a name "acmesolver" , but is that still the case? Does it matter?